File: src/main/java/ac/dnd/server/admission/infrastructure/web/dto/response/CurrentEventResponse.java
package ac.dnd.server.admission.infrastructure.web.dto.response;

import java.time.LocalDateTime;

import org.springframework.format.annotation.DateTimeFormat;

public record CurrentEventResponse(
	Long id,
	String name,
	@DateTimeFormat(pattern = "yyyy-MM-ddTHH:mm")
	LocalDateTime resultAnnouncementDateTime,
	Boolean isResultAnnounced
) {
}

File: src/main/kotlin/ac/dnd/server/admission/infrastructure/web/dto/response/CurrentEventResponse.kt
package ac.dnd.server.admission.infrastructure.web.dto.response

import org.springframework.format.annotation.DateTimeFormat
import java.time.LocalDateTime

data class CurrentEventResponse(
    val id: Long,
    val name: String,
    @field:DateTimeFormat(pattern = "yyyy-MM-ddTHH:mm")
    val resultAnnouncementDateTime: LocalDateTime,
    val isResultAnnounced: Boolean
)